db/migrate/20180706174552_create_rents.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rents do |t|
+      t.belongs_to :users, index: true
```

Comment:
Be careful with this, in the card says that this should be `t.references` and not `belongs_to`.
Sometimes, it works in the same way, not I'm not pretty sure that is the same.
Also, `user_id` and `book_id` should be foreign keys, They are keys from other table and this model "existence" depends on the other two models.